datarobot_batch_scoring/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.8.8'
+__version__ = '1.9.0'
```

Comment:
or go back to 1.8 if this runs like crap on a 2 core system. Our "script" might need to include system requirements now.